hibernate-core/src/main/java/org/hibernate/dialect/identity/DB2zIdentityColumnSupport.java

```diff
@@ -9,7 +9,7 @@
 /**
  * @author Andrea Boriero
  */
-public class DB2390IdentityColumnSupport extends DB2IdentityColumnSupport {
+public class DB2zIdentityColumnSupport extends DB2IdentityColumnSupport {
```

Comment:
Please also add a new class `DB2390IdentityColumnSupport` that extends `DB2zIdentityColumnSupport` and is marked as deprecated.